docker/rawr-batch/requirements.txt

```diff
@@ -27,3 +27,4 @@ zope.dottedname==4.2
 edtf==2.6.0
 mapbox-vector-tile==1.2.0
 boto3==1.7.10
+hanzidentifier==1.0.2
```

Comment:
I don't think we need this for raw tiles, but it shouldn't hurt.

Reply:
we don't.  I can leave it out if that's better